api/app/graphql/mutations/send_otp.rb

```diff
@@ -0,0 +1,28 @@
+module Mutations
+    class SendOtp < BaseMutation
+      argument :phone, String, required: true
+  
+      field :message, String, null: false
+  
+      def resolve(phone:)
+        # Find the user in the database based on their phone number
+        user = User.find_by(phone: phone)
+  
+        # Generate the OTP
+        otp = rand(2000..9000)
+  
+        # Send the OTP via Twilio
+        client = Twilio::REST::Client.new('AC6aa25af213eb27e64cb71b24a9453a47', '1cbc8aea593612a310687e9a69b68221')
```

Comment:
Remove the `account_ssid` and `auth_token` from the code.
Use environment variables instead.